doc/sphinx/proofs/automatic-tactics/logic.rst

```diff
@@ -166,6 +166,11 @@ Solvers for logic and equality
    You may want to use :tacn:`assert` to add some lemmas as
    hypotheses so that :tacn:`congruence` can use them.
 
+   .. tacv:: simple congruence
```

Comment:
We don't use `tacv` in the doc anymore (we just use indented `tacn` instead). Also you need to document the accurate full syntax, that is:
```suggestion
   .. tacv:: simple congruence {? @natural } {? with {+ @one_term } }
```

IIUC that the options accepted by `simple congruence` and `congruence` are the same.

Reply:
See also #14245 (which hasn't yet landed in `master` but should rather soon).

Reply:
Ok, I will change it to indented `.. tacn:: simple congruence {? @natural } {? with {+ @one_term } }`.
I wasn't sure because other `simple tac` tactics are documented somewhat incoherently.

Reply:
> other `simple tac` tactics are documented somewhat incoherently.

@Zimmi48 and I have been working to get the syntax correct for all tactics with comprehensible descriptions.  Any specific thoughts on how to reduce the incoherence?  Note that we've not yet updated the Basic Proof Writing/Tactics chapter yet, which has several of the "simple ..." tactics.  IIUC there's not a common, precise meaning for "simple" other than it's ordinary meaning, i.e. that it does something less that the non-simple version.